docs/update.xml

```diff
@@ -12,7 +12,7 @@
                               <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
                               <em:minVersion>38.0a1</em:minVersion>
                               <em:maxVersion>*</em:maxVersion>
-                              <em:updateLink>https://julienw.github.io/forms-extension/french_holiday_forms-latest.xpi</em:updateLink>
+                              <em:updateLink>https://julienw.github.io/forms-extension/french_holiday_forms-1.3.3-fx.xpi</em:updateLink>
```

Comment:
preparing for legacy mode: when releasing I'll manually add an update link for the new version that will then be using the new json-based update file.